dataset/src/main/scala/frameless/functions/NonAggregateFunctions.scala

```diff
@@ -158,6 +158,24 @@ trait NonAggregateFunctions {
     (implicit i0: CatalystCast[A, Double]): TypedAggregate[T, Double] =
       atan2(l, l.lit(r))
 
+  def sqrt[A, T](column: AbstractTypedColumn[T, A])
+                (implicit i0: CatalystCast[A, Double]): column.ThisType[T, Double] =
+    column.typed(sparkFunctions.sqrt(column.cast[Double].untyped))
+
+  def cbrt[A, T](column: AbstractTypedColumn[T, A])
+                (implicit i0: CatalystCast[A, Double]): column.ThisType[T, Double] =
+    column.typed(sparkFunctions.cbrt(column.cast[Double].untyped))
+
+  def exp[A, T](column: AbstractTypedColumn[T, A])
+                (implicit i0: CatalystCast[A, Double]): column.ThisType[T, Double] =
+    column.typed(sparkFunctions.exp(column.cast[Double].untyped))
+
+  def bround[A, B, T](column: AbstractTypedColumn[T, A])(
```

Comment:
Could you also add the doc?:

```
Returns the value of the column e rounded to 0 decimal places with HALF_EVEN round mode.
```

Reply:
Also it looks like there is a another version of bround that takes an extra argument `scale: Int`, do you think you could add that as well?

Reply:
sure i will try